extensions/import/src/wizard/flatFileWizard.ts

```diff
@@ -55,6 +56,8 @@ export class FlatFileWizard {
 		this.page2 = azdata.window.createWizardPage(constants.page2NameText);
 		this.page3 = azdata.window.createWizardPage(constants.page3NameText);
 		this.page4 = azdata.window.createWizardPage(constants.page4NameText);
+		this.createDerivedColumnButton = azdata.window.createButton(constants.createDerivedColumn);
```

Comment:
NIT: Can you move this to line https://github.com/microsoft/azuredatastudio/blob/a92ba424ac0fdfda1fa0e10d0b4a89d76af184b7/extensions/import/src/wizard/flatFileWizard.ts